src/CSV/Export.elm

```diff
@@ -7,7 +7,7 @@ module CSV.Export exposing (generate)
 -}
 
 import CSV.Template
-import Localized
+import Localized exposing (..)
```

Comment:
Please revert changes to this file. There do not appear to be substantive changes to this file. We avoid using `exposing (..)`. We usually choose module names and exposed names to go well together: `Localized.Element` is much clearer than `Element` IMHO.

Reply:
This might become quite "loud" because of the types I introduced. While it is easier to find the location of the definition, it forces a lot of repetition. I'll change it if you insist, but I prefer the slickness (TM).

Reply:
We side with https://github.com/stil4m/elm-analyse on this. Keeping this style-guide also reduce the size of the diff in this PR.